mod-orders/mod-orders.postman_collection.json

```diff
@@ -3567,6 +3589,153 @@
 						"description": "GET /orders/id requests that return 200"
 					},
 					"response": []
+				},
+				{
+					"name": "GET /orders/composite-orders - get orders list by empty query",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "eb2052a4-e388-4860-80be-ea96976bcf21",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "3bf00f6d-8798-4764-a256-2d360ea02876",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									" ",
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test(\"Validate that response contains default fields on creating an empty order\", function () {",
+									"    pm.expect(jsonData.purchase_orders).to.have.lengthOf(10);",
+									"});",
+									""
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "X-Okapi-Tenant",
+								"value": "{{xokapitenant}}"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "x-okapi-token",
+								"value": "{{xokapitoken}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{protocol}}://{{url}}:{{okapiport}}/orders/composite-orders",
+							"protocol": "{{protocol}}",
+							"host": [
+								"{{url}}"
+							],
+							"port": "{{okapiport}}",
+							"path": [
+								"orders",
+								"composite-orders"
+							]
+						},
+						"description": "GET /orders/composite-orders/id requests that return 200"
+					},
+					"response": []
+				},
+				{
+					"name": "GET /orders/composite-orders - get orders list by query",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "eb2052a4-e388-4860-80be-ea96976bcf21",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "3bf00f6d-8798-4764-a256-2d360ea02876",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									" ",
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test(\"Validate that response contains default fields on creating an empty order\", function () {",
+									"    pm.expect(jsonData.purchase_orders).to.have.lengthOf(7);",
```

Comment:
After local run against `folio-testing-backend01.aws.indexdata.com` I see that the check is environment dependent:

![image](https://user-images.githubusercontent.com/43410167/51604681-93c2ae00-1f1e-11e9-8443-763acd7184d6.png)